examples/tutorials/advanced/grid_equalization.py

```diff
@@ -0,0 +1,222 @@
+"""
+Performing grid histogram equalization
+--------------------------------------
+The :meth:`pygmt.grdhisteq.equalize_grid` function creates a grid with
+statistics based on a cumulative distribution function.
+"""
+# sphinx_gallery_thumbnail_number = 3
+
+import pygmt
+
+###############################################################################
+# Load sample data
+# ----------------
+# Load the sample Earth relief data for a region around Yosemite valley
+# and use :meth:`pygmt.grd2xyz` to create a :class:`numpy.ndarray` with the z
+# values.
+region = [-119.825, -119.4, 37.6, 37.825]
```

Comment:
`region` is only used once in this script at line 19, so no need to define this variable.